autogpt_platform/backend/backend/util/service.py

```diff
@@ -175,6 +178,8 @@ def cleanup(self):
         if self.use_redis:
```

Comment:
Is it valid to use both rabbit and redis?

Reply:
Yes